python/pipeline/preprocess.py

```diff
@@ -13,10 +14,15 @@
 from . import config
 from distutils.version import StrictVersion
 from .utils import galvo_corrections
+from .experiment import Session, Scan
+import imreg_dft as ird
+
+# import caiman.source_extraction.cnmf as cnmf
 
 assert StrictVersion(dj.__version__) >= StrictVersion('0.2.8')
 
 schema = dj.schema('pipeline_preprocess', locals())
+schema.spawn_missing_classes()
```

Comment:
Why? We want to load all classes and then replace those for which we have actual implementation. Otherwise we'll overwrite the existing classes.